examples/advanced/llm_hf/requirements.txt

```diff
@@ -0,0 +1,8 @@
+nvflare
+torch
+datasets
+tensorboard
+transformers
+peft
+trl
```

Comment:
Seems we are missing the trl license as well.

Reply:
added peft and trl